projects/core/src/cart/facade/cart.service.ts

```diff
@@ -85,49 +73,35 @@ export class CartService {
           new fromAction.LoadCart({
             userId: this.cartData.userId,
             cartId: 'current',
+            details: true,
```

Comment:
Why now we need to call for detailed cart?

Reply:
I did it for now to simplify things. Always load everything, but I am thinking that if we want everyone to use `getActive` method we should allow passing an option for details. And use information from there to load details or not. What do you think?

Reply:
Looks good idea. We can also postpone adding this option for `getActive` after 1.0. Please create a ticket then.

Reply:
Done

Reply:
Reference: https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/3241